src/style.css

```
@@ -0,0 +1,24 @@
+.body {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+.h1 {
+  font-size: 17px;
+}
+
+.h2 {
+  font-size: 15px;
+  font-weight: normal;
+  padding: 1em 0;
+}
+
+.link {
```

```suggestion
.colored-link {
```
давай додамо трохи конкретики